packages/shared/src/components/auth/ChangePasswordForm.tsx

```diff
@@ -94,6 +94,7 @@ function ChangePasswordForm({
           inputId="password"
           type="password"
           name="password"
+          className={{ container: 'w-full' }}
```

Comment:
Do we have this anywhere else in the code?
Wondering if we need to apply it in more places maybe

Reply:
Let me verify quickly.

Reply:
Last form we haven't seen. Seems to be working as intended 🚀 

![Screenshot 2023-09-12 at 9 39 21 PM](https://github.com/dailydotdev/apps/assets/13744167/55bc1a86-e955-491a-af1c-9a0b6ec28755)

Reply:
Alright thanks for checking, good to go